pkgs/development/interpreters/python/hooks/setuptools-build-hook.sh

```diff
@@ -12,7 +12,9 @@ setuptoolsBuildPhase() {
         args+="$setupPyGlobalFlags"
     fi
     if [ -n "$enableParallelBuilding" ]; then
-        setupPyBuildFlags+=" --parallel $NIX_BUILD_CORES"
+        if [ -v "$setuptools_has_parallel" ]; then
```

Comment:
I think `-v` requires a variable name like `setuptools_has_parallel`, not the expansion like `"$setuptools_has_parallel"`:

```
$ a=; [ -v "$a" ] && echo ok || echo unset
unset
$ a=1; [ -v "$a" ] && echo ok || echo unset
unset
```

If we use the variable name it's almost ok, but the problem is that empty string assignment is still an assignment:

```
$ a=; [ -v a ] && echo ok || echo unset
ok
$ a=1; [ -v a ] && echo ok || echo unset
ok
```

Reply:
Why are we not using -n like in the other vars?

Reply:
Switched to `-n`. I intended to use `-n` initially. No idea how `-v` slipped in.